website/cue/reference/components/sinks.cue

```diff
@@ -236,7 +236,7 @@ components: sinks: [Name=string]: {
 													type: string: {
 														examples: [
 															"""
-															["{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }"]
+															[{ "type": "record", "name": "log", "fields": [{ "name": "message", "type": "string" }] }]
```

Comment:
I think the schema actually does need to be defined as a string containing JSON 🤔

Reply:
If you use the following configuration:
```
encoding.avro.schema = "[\"{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }\"]"
```
and run Vector, you will get the following error:
```
2022-11-08T21:08:38.353507Z ERROR vector::topology: Configuration error. error=Sink "kafka": Failed building Avro serializer: Failed to parse schema from JSON
```

But if you use the following config:
```
encoding.avro.schema = "[{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }]"
```
Vector initializes Avro serializer successfully.

Reply:
If you will check the code, you can find that as a string is treated the whole `encoding.avro.schema` field. However, probably as a valid Avro schema will be the following string:
```
encoding.avro.schema = "{ \"type\": \"record\", \"name\": \"log\", \"fields\": [{ \"name\": \"message\", \"type\": \"string\" }] }"
```
without `[]` brackets.

Reply:
Doh, yeah, I see, it is double quoted. I missed that this is already in a `"""` block.